woocommerce/woocommerce-android
# Feature/8092 add missing profiler steps

Closes: #8092


### Description
This PR is a continuation of[ this other](https://github.com/woocommerce/woocommerce-android/pull/8009). It adds the missing profiler steps:
- Commerce Journey Step
- Ecommerce platforms step

**Couple things to keep in mind:** 

- Profiler steps are still under feature flag and only available for debug builds as we are still discussing on the options that should be displayed. This PR is also missing localization for the displayed options. Once we have the definitive list of options to be displayed we can proceed to add them as strings to get them localized. More context here p1672316116056879-slack-C045CUK1Y3U

- This PR doesn't include saving the profiler options in backend because the API is still WIP.

- The scope of these changes don't include adding the Country selection screen. So right now after the profiler steps are done, user will land on domain picker screen.

### Testing instructions
- Trigger store creation flow
- Navigate through profiler steps selecting different option and verifying the navigation between complies with the expected design: 

![Screenshot 2022-12-30 at 20 23 07](https://user-images.githubusercontent.com/2663464/210105445-c5478f8a-6281-4237-9695-90a03154f484.png)

- Check that selected state is correctly displayed in dark mode, following this design:

![Screenshot 2022-12-30 at 20 25 12](https://user-images.githubusercontent.com/2663464/210105593-af23ef38-e4f4-4536-b9e1-f9edd0cad1ce.png)

I will be AFF during the next week. So, if there's anything blocking I won't be able to fix it until the 9th January. Feel free to apply any changes you consider and resume work from here if needed @0nko
